scripts/apps/authoring/authoring/directives/AuthoringHeaderTansaDirective.js

```diff
@@ -0,0 +1,21 @@
+AuthoringHeaderTansaDirective.$inject = ['$rootScope'];
```

Comment:
Please explain what this is and what it does. It's hard to review it because I don't understand the scope of this directive. Also, why are we binding the directive's node to the scope? Why is this needed?